app/assets/stylesheets/errors.scss

```diff
@@ -0,0 +1,14 @@
+#error_explanation {
```

Comment:
Check out CSS naming conventions: https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849